airbyte-integrations/connectors/source-mixpanel/source_mixpanel/manifest.yaml

```diff
@@ -210,6 +219,17 @@ definitions:
         http_method: POST
       paginator:
         $ref: "#/definitions/paginator"
+      record_selector:
+        $ref: "#/definitions/selector"
+        record_filter:
+          condition: |
+            {%- set partition_state = namespace(value=config.start_date) -%}
```

Comment:
@artem1205 FYI - flagging that we need to make sure the our semi-incremental implementation works for substreams.

Reply:
@midavadim is support for semi-incremental required for the bug fix or could this be released in 3.0.1? reason is that `3.0.0` will not be revertible so I'd prefer keeping the scope of the change as small as possible

Reply:
@girarda  "semi-incremental" - WIthout filtering each incremental syncs returns all records. it is implemented in order to support proper 'incremental' sync which existed in python version of this connector (2.2.0). When we were discussing this change, it was agreed that we must implement the same behavior as it was before low code otherwise it is treated as another breaking change. Please correct me if this is a wrong statement.